src/components/formio/datamap.tsx

```diff
@@ -30,6 +38,9 @@ export const DataMap: React.FC<DataMapProps> = ({name, valueComponent, keyLabel
               const newValue = {...value, [item.key]: event.target.value};
               setValue(newValue);
             },
+            'aria-label': ariaLabelMessage
+              ? intl.formatMessage(ariaLabelMessage, {key: item.key})
+              : null,
```

Comment:
```suggestion
            'aria-label': ariaLabelMessage
              ? intl.formatMessage(ariaLabelMessage, {key: item.key})
              : undefined,
```
using `undefined` makes it as if the prop was not passed at all, where-as `null` treats it that the prop *was* passed.